dev/fattest.simplicity/src/com/ibm/websphere/simplicity/BeansAsset.java

```diff
@@ -42,32 +42,72 @@
  *              .addAsWebInfResource(beans, "beans.xml");
  * </code>
  * </pre>
+ *
```

Comment:
Please raise an issue rather than check in TODOs in code.

Reply:
Good point - https://github.com/OpenLiberty/open-liberty/issues/16057